test/index.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const Bookshelf = require('bookshelf')(require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  }
+}));
```

Comment:
Ahh yes I can see where it might get annoying in the future if the schema or data needs to change. Is there an easy way to setup the SQLite DB in a before hook in the tests?